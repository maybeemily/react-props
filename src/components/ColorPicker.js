import React from 'react';

export default function ColorPicker() {

  const style1 = {
    height: '30px',
    width: '30px',
    'background-color': 'pink'
  };
  const style2 = {
    height: '30px',
    width: '30px',
    'background-color': 'yellow'
  };
  const style3 = {
    height: '30px',
    width: '30px',
    'background-color': 'powderblue'
  };
  return (
    <ul>
      <li><span style={style1}>PINK</span></li>
      <li><span style={style2}>YELLOW</span></li>
      <li><span style={style3}>BLUE</span></li>
    </ul>
  );

}
