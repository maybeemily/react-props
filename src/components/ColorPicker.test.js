import React from 'react';
import { shallow } from 'enzyme';
import ColorPicker from './ColorPicker';

describe('Color Picker tests', () => {
  it('renders the color picker', () => {
    const wrapper = shallow(<ColorPicker />);
    expect(wrapper).toMatchSnapshot();
  });
});
