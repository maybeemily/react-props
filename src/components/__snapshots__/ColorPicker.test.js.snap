// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Picker tests renders the color picker 1`] = `
<ul>
  <li>
    <span
      style={
        Object {
          "background-color": "pink",
          "height": "30px",
          "width": "30px",
        }
      }
    >
      PINK
    </span>
  </li>
  <li>
    <span
      style={
        Object {
          "background-color": "yellow",
          "height": "30px",
          "width": "30px",
        }
      }
    >
      YELLOW
    </span>
  </li>
  <li>
    <span
      style={
        Object {
          "background-color": "powderblue",
          "height": "30px",
          "width": "30px",
        }
      }
    >
      BLUE
    </span>
  </li>
</ul>
`;
